common.pri

```diff
@@ -40,3 +40,5 @@ OBJECTS_DIR = ../../build/.obj
 MOC_DIR = ../../build/.moc
 RCC_DIR = ../../build/.rcc
 UI_DIR = ../../build/.ui
+
+LIBS += -lSimpleAmqpClient -lrabbitmq
```

Comment:
If you update the README to include brain dead instructions, updating travis will be easy.